app/templates/main/catalogue.html

```diff
@@ -2,42 +2,170 @@
-    <div class="tna-column tna-column--full">
-      {{ tnaBreadcrumbs({
-        'items': [
-          {
-            'text': 'Home',
-            'href': '/'
-          }
-        ],
-        'structuredData': True,
-        'classes': 'tna-!--padding-vertical-s'
-      }) }}
+    <form id="search-form" method="get" class="tna-column tna-column--full tna-!--margin-bottom-m">
+      <div class="tna-search-field ">
+  <div class="tna-form__group ">
+    <div class="tna-form__group-contents">
+      <h1 class="tna-form__heading tna-form__heading--l">
+        <label class="tna-form__label" for="search">
+          Search our catalogue
+        </label>
+      </h1>
+      <p>Search by keyword, place, person, or catalogue reference</p>
+    </div>
+    <div class="tna-search-field__fields">
+      <input type="search" id="search" class="tna-search-field__input " name="q" value="tall" spellcheck="false">
+      <button type="submit" class="tna-button tna-search-field__button">
+        Search
+        <i class="fa-solid fa-fw fa-arrow-right tna-!--hide-on-tiny" aria-hidden="true"></i>
+      </button>
     </div>
   </div>
-{% endblock %}
+  
+</div>
+
+    <input type="hidden" name="display" value="">
+    </form>
+    
+  </div>
+</div>
+
+<div class="choose-section tna-!--padding-top-s">
+
+  <div class="tna-accordion tna-container" data-multiselectable="false" data-module="tna-accordion">
+    <div class="tna-accordion__item" data-isopen="true">
+      <h3 class="tna-accordion__heading tna-heading-s">
+        Find records by subject
+      </h3>
+      <div class="tna-accordion__body" id="accordion-demo-content-1">
+        <div class="tna-container tna-!--margin-top-m">
+  
+  
+  
+          <div class="tna-column tna-column--width-1-3 tna-column--width-1-2-small tna-column--full-tiny">
+            <strong>Select a letter</strong>
+            <div class="a-z">
+            
+              <ul class="a-z">
+                <li><a href="/tags/browse/A">A</a></li><li><a href="/tags/browse/B">B</a></li><li><a href="/tags/browse/C">C</a></li><li><a href="/tags/browse/D">D</a></li><li><a href="/tags/browse/E">E</a></li><li><a href="/tags/browse/F">F</a></li><li><a href="/tags/browse/G">G</a></li><li><a href="/tags/browse/H">H</a></li><li><a href="/tags/browse/I">I</a></li><li><a href="/tags/browse/J">J</a></li><li><a href="/tags/browse/K">K</a></li><li><a href="/tags/browse/L">L</a></li><li><a href="/tags/browse/M">M</a></li><li><a href="/tags/browse/N">N</a></li><li><a href="/tags/browse/O">O</a></li><li><a href="/tags/browse/P">P</a></li><li><a href="/tags/browse/Q">Q</a></li><li><a href="/tags/browse/R">R</a></li><li><a href="/tags/browse/S">S</a></li><li><a href="/tags/browse/T">T</a></li><li><a href="/tags/browse/U">U</a></li><li><a href="/tags/browse/V">V</a></li><li><a href="/tags/browse/W">W</a></li><li><a href="/tags/browse/X">X</a></li><li><a href="/tags/browse/Y">Y</a></li><li><a href="/tags/browse/Z">Z</a></li>
```

Comment:
`/tags/browse/X` doesn't exist yet, does it?

If not, can we just have these as `href="#x"` for now?

Reply:
I've updated those links now and reformatted the html to be more readable.